packages/next/plugins/with-nx.ts

```diff
@@ -221,8 +226,8 @@ function withNx(
             ? joinPathFragments(workspaceRoot, projectDirectory)
             : workspaceRoot,
           projectDirectory,
-          _nextConfig.nx?.fileReplacements || options.fileReplacements,
-          _nextConfig.nx?.assets || options.assets
+          _nextConfig.nx?.assets || options.assets,
+          _nextConfig.nx?.fileReplacements || options.fileReplacements
```

Comment:
@ndcunningham The order is incorrect here.  `createWebpackConfig()` expects assets to be the last argument.  Please see here https://github.com/nrwl/nx/blob/6d83ae2fb1806dbc3f6505cdd7d682ae4b662169/packages/next/src/utils/config.ts#L19